_search-plugins/searching-data/filtering-search-results.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: default
+title: Filter results
+parent: Searching data
+nav_order: 21
+redirect_from:
```

Comment:
Remove line 6 and 7. Redirects are only needed if a previous version of the same file was in a different directory/had a different name.